extensions/ui/org.eclipse.smarthome.ui.paper/package.json

```diff
@@ -3,7 +3,14 @@
   "version": "0.8.0",
   "private": true,
   "dependencies": {
-    "bower": "^1.4.1",
+    "angular": "^1.4.9",
```

Comment:
You should stick to the versions we had previously (1.3.13, 0.10.0 for angular-material, 3.3.2 for bootstrap, etc.) as these are the ones where we have IP approval.